src/analytics/private/athena-saved-queries.ts

```diff
@@ -16,16 +16,15 @@ import { CfnNamedQuery } from 'aws-cdk-lib/aws-athena';
 import { Construct } from 'constructs';
 import { AthenaBuiltInQueries } from './athena/query';
 import { getSqlContent } from './utils';
+import { AthenaAnalyticsStackProps } from '../parameter';
 
-export interface AthenaSavedQueryProps {
-  readonly databaseName: string;
-  readonly workGroup: string;
-  readonly eventTable: string;
-}
+export type AthenaSavedQueryProps = AthenaAnalyticsStackProps;
 
 export type MustacheParamType = {
```

Comment:
You might define via typescript's Omit/Partial/Pick from `AthenaSavedQueryProps`.

Reply:
fixed